src/Models/ResponseModels/StepResponseModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SolutionsService.Models.ResponseModels
+{
+    public class StepResponseModel
+    {
+        public Guid Id { get; set; }
+        public string CoverImage { get; set; }
```

Comment:
Should this be a string?

Reply:
What else?

Reply:
There's an Image class but that's probably not storable in a database. I presume this is just the link? Might want to reflect that it's a link to the image in the name e.g. `CoverImageURL`